frame/staking-rewards/README.md

```diff
@@ -1,116 +1,221 @@
-- the penalty beneficiary, the account where the penalty are going to be
-  transferred to
+The staking pool has a configuration explaining which staking positions other accounts can have.
 
-Once this extrinsic has been dispatched, let's assume we want to reward `PICA`
-stakers, the mapping will be `PICA => ([(WEEK, 0.5), (MONTH, 0.8), (TWOMONTH,
-1.0)], [PICA, BTC, ETH])`. Meaning that users will be able to stake `PICA` for
-either a `WEEK` for a reward multiplier of `0.5` or a `MONTH` etc... The
-incentive for the users to stake will be the fact that they will be able to
-harvest `PICA` or `BTC` or `ETH` whenever a reward distribution occured.
+User transfers staked assets into the staking pool and choose what position he wants to take. She becomes a `staker`.
+
+Protocol owner increases total rewards share as time goes.
+
+Anybody can mint and transfer rewarded assets into pool. This process can be automated.
+
+A `staker` may leave a position, claim rewards, stake more, etc.
+Or she can make [a position to be fNFT](../nft/README.md) and use it as an instrument in other protocols.
+
+
+## Protocol
+
+The `staking protocol` is configured with ED for having pools and positions on-chain.  
+
+## Pool
+
+A pool is where users put tokens and get shares of rewards or other powers. 
+
+Any protocol can have one or more pools. The main characteristic of a pool is staked asset identifier(token).
+
+Each pool is governed through a protocol that created it.
+
+### Configuration
+
+To allow the users to stake an asset, we first need to `configure` the pool. 
+
+Configuration of a pool defines:
+
+Can/must user time lock tokens to get, possibly increased, `share`. What is the penalty for early unlock if there is a lock
+  
+If share decays with time until lock ends and extending time starts from zero elapsed.
+
+Or if the share does not decay with time and extending time subsumes old-time already passed in a lock (user gets smaller remaining time with higher share).
+
+Owner account of the pool. Usually other protocol or its "owner".
+
+The inflation rate "grows" rewards as time goes. It can be dynamically updated later.
+  
+**Example**,
+
+The set of staking durations along with their reward multiplier (e.g. `[(WEEK,
+  0.5), (MONTH, 0.8), (TWO_MONTH, 1.0)]`).
+The early unstake penalty can be set `0.7`.
+
+And asset to be PICA.
+
+Above means that users will be able to stake `PICA` for
+either a `WEEK` for a reward multiplier of `0.5` or a `MONTH` etc.
+The incentive for the users to stake will be the fact that they will be able to
+harvest `PICA` or `BTC` or `ETH` whenever a reward distribution occurred.
+
+See details in `stake positions` operation on how these values influence users.
+
+See pool rewards mechanics on how rewards can be distributed.
+
+### Rewarding
+
+Only the pool owner can transfer the rewards of new assets into a `pool rewards account`. But anybody can transfer rewards of assets that previously were rewarded.
```

Comment:
Is there a particular reason for this condition?

Reply:
pool created. anybody spammed with shitcoints. pool is dead.

Reply:
Reward transfers are generally done by protocols which want to reward certain pools because they want to incentivize using their own token(just distributing tokens to encourage use), so first transfer by owner rule does not work. If you are talking about extrinsic methods, we don't really have a requirement for that. I can suggest getting rid of this sentence or modifying it as,

```suggestion
Only the pool owner can transfer the rewards of new assets into a `pool rewards account` through any extrinsic methods.
```